### DISCUSSION TITLE
How to mutate state in zustand log middleware

### DISCUSSION THREAD

**Discussion by markferdinandsoriano** [pmndrs/zustand#1708] [General]
![Screenshot from 2023-03-22 19-42-41](https://user-images.githubusercontent.com/126935039/226893866-f649673a-3d76-451a-a5b3-19ec22c7ab59.png)

since using the get() function i can see the states update realtime, but how can i mutate the state inside this log middleware in zustand, Please help me, i didn't find any solutions thru internet.

**Comment by dai-shi** [comment]
Doesn't `set(...)` work?

**Reply by markferdinandsoriano** [reply]
yes sir also the set doesn't work so. I don't know how to mutate and set again the data inside this middleware log

**Reply by dai-shi** [reply]
Maybe you want to create a minimal reproduction with https://codesandbox.io/s/react-new to get help from someone.
